src/targets/registry.ts

```diff
@@ -270,12 +269,12 @@ export class RegistryTarget extends BaseTarget {
 
     await mapLimit(artifacts, MAX_DOWNLOAD_CONCURRENCY, async artifact => {
       const fileData = await this.getArtifactData(artifact, version, revision);
-      if (!_.isEmpty(fileData)) {
+      if (Object.keys(fileData).length > 0) {
```

Comment:
_nit_: This could be put behind a utility function to show clear intent. 

If someone wants to optimize, they can then easily switch this out for an implementation that doesn't require collecting keys (FWIW, misses `hasOwnProperty` check):


```js
function isEmpty(obj) {
  for (key in obj) return true;
  return false;
}
```

Reply:
Can do.